### ISSUE TITLE
require_once(/home/sorpresa/public_html/catalog/controller/payment/mollie/helper.php): failed to open stream

### ISSUE THREAD

**Issue by tyhawk** [mollie/OpenCart#256]
After updating my Mollie install on OpenCart 3.0.3.6 (which worked fine) to version 10.1.1, the admin section of my website worked fine when I checked if all the Mollie settings where still correct. After that, I clicked on the dashboard and got this error:
```
Warning: require_once(/home/sorpresa/public_html/catalog/controller/payment/mollie/helper.php): failed to open stream: No such file or directory in /home/sorpresa/public_html/vqmod/vqcache/vq2-admin_controller_common_dashboard.php on line 18
```
I checked and the file catalog/controller/payment/mollie/helper.php isn't there:
```
[myuser public_html]$ ls catalog/controller/payment/mollie/
base.php
```
It also isn't in the zip file, so it seems to be absent completely.

**Upgrade steps I took:**
1. I downloaded the zip file from this Github project.
2. I unzipped it.
3. I uploaded the admin and catalog folders (recursively) to the public_html folders on my webserver with scp.

Before I took these steps, I was presented with a warning that there was an update for the Mollie plgin, including an "update now" link. When I clicked on it, my https://myweb.shop/admin/ URL became a blank page. I had to revert to my daily backup to fix this.

To resolve the issue in this bug report, I had to revert to my daily backup a second time. I don't think updating a plugin should cause my admin panel to break like this.

**Comment by QualityWorks** [comment]
It seems like the old modification is still there. please make sure to refresh the modifications after updating.

**Comment by QualityWorks** [comment]
have you been able to resolve this? or do you need help?

**Comment by tyhawk** [comment]
I haven't had time for a second attempt, but I will try again soon. Thanks for the reply.

**Comment by tyhawk** [comment]
Second attempt: I removed all the files of my current Mollie install (which is a task that took me quite a while) and copied both the `admin` and `catalog` folders from the zip file into my public_html folder. In OpenCart, I went into Extensions > Modifications and hit reload. I then went to Dashboard and I still got the error.
```
Warning: require_once(/home/sorpresa/public_html/catalog/controller/payment/mollie/helper.php): failed to open stream: No such file or directory in /home/sorpresa/public_html/vqmod/vqcache/vq2-admin_controller_common_dashboard.php on line 18
```
I again had to revert to my backup to get the Dashboard working again.
